src/pool.py

```diff
@@ -0,0 +1,14 @@
+from .client import DiscordClient
+
+
+class Pool:
+    def __init__(self, tokens: list[str]) -> None:
+        self.tokens = tokens
+        self.cursors: list[DiscordClient] = [
+            DiscordClient.login(token) for token in tokens
+        ]
+
+    async def sort(cls):
+        # TODO: implement more sorting methods
+        self = cls()
```

Comment:
```suggestion
    async def sort(self):
        # TODO: implement more sorting methods
```
Just old code from when it was a classmethod